src/app/routes/admin-forms.server.routes.js

```diff
@@ -189,7 +198,89 @@ module.exports = function (app) {
   app
     .route('/:formId([a-fA-F0-9]{24})/adminform')
     .get(withUserAuthentication, AdminFormController.handleGetAdminForm)
-    .put(authActiveForm(PermissionLevel.Write), adminForms.update)
+    .put(
+      withUserAuthentication,
+      celebrate({
```

Comment:
small request: can we extract this into a different file? I know I've said the exact opposite before by asking to inline the `celebrate` middleware, but I feel in this case the middleware is so long that all the other tests are hard to see at a glance

Reply:
Instead of extracting into a different file, i'll just extract the validator to a constant and put it at the top of the file, does that sound good?

Reply:
Implemented in 40d8a03.